tests/test_views.py

```diff
@@ -238,18 +238,29 @@ def test_invalid_shapefile(self):
         self.assertIsNotNone(response.data['errors'])
         self.assertEqual(response.data['status'], Boundary.StatusTypes.ERROR)
 
-    # TODO: this test started failing for some reason -- need to figure out why
```

Comment:
This test was failing because `rest_framework_gis` was not included in `INSTALLED_APPS`, which produces esoteric downstream errors.